client/blocks/reader-excerpt/index.jsx

```diff
@@ -18,10 +19,12 @@ const ReaderExcerpt = ( { post, isDiscover } ) => {
 
 	return (
 		<AutoDirection>
-			<div
-				className="reader-excerpt"
-				dangerouslySetInnerHTML={ { __html: excerpt } } // eslint-disable-line react/no-danger
-			/>
+			<Emojify>
```

Comment:
Yah, I need to play with it a bit more. I'm occasionally seeing React throw a warning that an object is begin passed to Emojify, instead of a string.